src/i18n/strings/bg.json

```diff
@@ -2461,7 +2461,7 @@
     "Spaces": "Пространства",
     "Check your devices": "Проверете устройствата си",
     "%(deviceId)s from %(ip)s": "%(deviceId)s от %(ip)s",
-    "This homeserver has been blocked by it's administrator.": "Този Home сървър е бил блокиран от администратора му.",
+    "This homeserver has been blocked by its administrator.": "Този Home сървър е бил блокиран от администратора му.",
```

Comment:
we cannot change strings in these files. Only files modified by `yarn i18n` are possible.

Reply:
I don't understand what files must not be changed and why.

Reply:
done